src/components/experiment-tracking/run-details-modal/run-details-modal.test.js

```diff
@@ -5,6 +5,24 @@ import { configure, mount, shallow } from 'enzyme';
 
 configure({ adapter: new Adapter() });
 
+// Mocked methods
+
+const mockReset = jest.fn();
+const mockUpdateRunDetails = jest.fn();
+
+jest.mock('../../../apollo/mutations', () => {
+  return {
+    useUpdateRunDetails: () => {
+      return {
+        reset: mockReset,
+        updateRunDetails: mockUpdateRunDetails,
+      };
+    },
+  };
```

Comment:
Now that we have set up the mock for this mutation function, let's add the test that actually calls the `updateRunDetails` function ( which is called from `onApplyChanges` ), as well as mocking and testing the error state for the error message (i.e testing the existence of the `run-details-modal-error-wrapper` class) - otherwise there isn't any point of setting up this mock function here.